pyproject.toml

```diff
@@ -12,13 +12,14 @@ readme = "README.md"
 authors = ["Statnett Datascience <Datascience.Drift@Statnett.no>"]
 repository = "https://github.com/statnett/cimsparql.git"
 include = ["pkg_data/*"]
+exclude = ["tests/*"]
 
 [tool.poetry.dependencies]
 SPARQLWrapper = "*"
 StrEnum = "*"
 pandera = "*"
 pandas = "*"
-python = "^3.9"
+python = "^3.9, <3.12"
```

Comment:
This will cause problems in any project that wants to use cimsparql but has specified the python dependency as for example ^3.9 (which is equivalent to >=3.10,<4.0.

```
The current project's Python requirement (>=3.10,<4.0) is not compatible with some of the required packages Python requirement:
  - cimsparql requires Python ^3.9, <3.12, so it will not be satisfied for Python >=3.12,<4.0
```

Reply:
It seems like pandera does not support anything above 3.11

Reply:
Ok. Added an issue on pandera about that.